lua/catppuccin/groups/integrations/mini.lua

```diff
@@ -8,17 +8,99 @@ function M.get()
 
 	local indentscope_color = O.integrations.mini.indentscope_color
 	return {
+		MiniAnimateCursor = { style = { "reverse", "nocombine" } },
+		MiniAnimateNormalFloat = { link = "NormalFloat" },
+
+		MiniClueBorder = { link = "FloatBorder" },
+		MiniClueDescGroup = { link = "DiagnosticFloatingWarn" },
+		MiniClueDescSingle = { link = "NormalFloat" },
+		MiniClueNextKey = { link = "DiagnosticFloatingHint" },
+		MiniClueNextKeyWithPostkeys = { link = "DiagnosticFloatingError" },
+		MiniClueSeparator = { link = "DiagnosticFloatingInfo" },
+		MiniClueTitle = { link = "FloatTitle" },
+
 		MiniCompletionActiveParameter = { style = { "underline" } },
 
 		MiniCursorword = { style = { "underline" } },
 		MiniCursorwordCurrent = { style = { "underline" } },
 
+		MiniDepsChangeAdded = { link = "diffAdded" },
+		MiniDepsChangeRemoved = { link = "diffRemoved" },
+		MiniDepsHint = { link = "DiagnosticHint" },
+		MiniDepsInfo = { link = "DiagnosticInfo" },
+		MiniDepsMsgBreaking = { link = "DiagnosticWarn" },
+		MiniDepsPlaceholder = { link = "Comment" },
+		MiniDepsTitle = { link = "Title" },
+		MiniDepsTitleError = { bg = C.red, fg = C.base },
+		MiniDepsTitleSame = { link = "DiffText" },
+		MiniDepsTitleUpdate = { bg = C.green, fg = C.base },
+
+		MiniDiffSignAdd = { fg = C.green },
+		MiniDiffSignChange = { fg = C.yellow },
+		MiniDiffSignDelete = { fg = C.red },
+		MiniDiffOverAdd = { link = "DiffAdd" },
```

Comment:
Just wondering: shouldn't these linked highlights be linked inside `mini.nvim`? So that there only need to be the necessary ones that must be manually colored to look correct.

Reply:
All of these links *are* done in 'mini.nvim', but those are default links: they take effect only if color scheme doesn't define the corresponding highlight groups.

Not having those explicit links will work if this is the only loaded color scheme. But it might not fully work if `:colorscheme catppuccin` is executed *after* applying color scheme that overrides some highlight groups.

Reply:
Got it